apis/database/v1beta1/dbsubnetgroup_types.go

```diff
@@ -151,31 +140,17 @@ type DBSubnetGroupList struct {
 	Items           []DBSubnetGroup `json:"items"`
 }
 
-// UpdateExternalStatus updates the external status object, given the observation
-func (b *DBSubnetGroup) UpdateExternalStatus(observation rds.DBSubnetGroup) {
-
-	subnets := make([]Subnet, len(observation.Subnets))
-	for i, sn := range observation.Subnets {
-		subnets[i] = Subnet{
-			SubnetID:     aws.StringValue(sn.SubnetIdentifier),
-			SubnetStatus: aws.StringValue(sn.SubnetStatus),
-		}
-	}
+// DBSubnetGroupObservation is the representation of the current state that is observed
```

Comment:
nitpick: would you mind moving this up above `DBSubnetGroupStatus` where `DBSubnetGroupExternalStatus` is currently?